test/api/python/provisioner/test_r2_hare.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import pytest
+from unittest.mock import patch, MagicMock
+
+from provisioner.hare import r2_check_cluster_health_status
+
+
+@pytest.mark.unit
+def test_cluster_health_status_over_cli():
+    """
+    Test for Provisioner wrappers over HA CLI commands.
+
+    Returns
+    -------
+
+    """
+    with patch('provisioner.hare.cmd_run', MagicMock()) as mh:
+        mh.side_effect = Exception("Some Exception")
+
+        assert r2_check_cluster_health_status() is False
+
+        mh.reset_mock(side_effect=True)
+        mh.return_value = dict()
+
+        assert r2_check_cluster_health_status() is False
+
+        mh.return_value = dict(res=True)
+
+        assert r2_check_cluster_health_status() is True
+
+
+@pytest.mark.skip(reason='EOS-20624')
```

Comment:
Could you please add a comment as to why we skip case of `EOS-20624`? May be a bit elaborative reason if possible.

Reply:
Done, I added a proper comment here
```
@pytest.mark.skip(reason='EOS-20624: Hare Python API is not ready yet')
```